src/gdb/gdb.c

```diff
@@ -52,6 +52,9 @@ static void write_sr(int sr);
 static unsigned char *mem_to_hex(const void *mem_,
 	unsigned char *buf, int count);
 static void read_sr(int sr);
+static unsigned char *hex_to_mem(const unsigned char *buf, void *mem_,
+							int count);
+
```

Comment:
It looks like the functions that need more than one line are poorly indented. The second line should start exactly where the first parameter from the first line starts.

So, "int count" should align with "const unsigned...". Start the indentation with tabs then add space for correctly alignment.

Reply:
@dbaluta, thanks for suggestions.
One caveat though, the checkpatch.pl which you referred to has indeed listed some lines but did it after the commit was made. Moreover printed it starting with *green* CHECK message which clearly suggest the lines is OK...

Reply:
@mrajwa you're welcome. 

Yes, the checkpatch only points the CHECK message after commit is made because --strict parameter is missing from pre-commit hook. @plbossart  is this intended?

CHECK is printed with green because it is not an ERROR, not even a WARNING. Anyhow, in order to obey the coding style of the sof project I think you should really fix the indentation. It looks unnatural comparing with the rest of the project.

For, example look in src/drivers/dw/gpio.c at gpio_write function. You should follow the same convention when indenting the functions in your code :)

Reply:
No doubt the code will be aligned I have overlooked it and green checkpatch message did not wake me up.